### PR TITLE
Issue-4930 unified android build pipeline

### PR DESCRIPTION
**Pull request by britzl**
Make sure to generate a debug.keystore if no keystore is provided when bundling for Android

### CODE REVIEW

File: com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidBundler.java [hunk 1]
```diff
@@ -109,27 +113,66 @@ private static Result exec(String... args) throws IOException {
...
+    */
+    private static String getKeystore(Project project) throws IOException {
+        String keystore = project.option("keystore", "");
+        if (keystore.length() == 0) {
+            keystore = "debug.keystore";
+            String keystoreAlias = "androiddebugkey";
+            String keystorePassword = "android";
+            String keystorePasswordFile = "debug.keystore.pass.txt";
+            if (!new File(keystore).exists()) {
+                Result r = exec("keytool",
```
Comment (JCash) [inline com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidBundler.java:130]: Is this tool available on Linux/Windows?

**Review by JCash — DISMISSED** [review]
Approved if the tool is available on all platforms.

File: com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidBundler.java (hunk 1 above)
Comment (britzl) [reply com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidBundler.java]: Added method to use System.getProperty("java.home") or System.getenv("JAVA_HOME") when running keytool.

File: com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidBundler.java [hunk 2]
```diff
@@ -109,30 +124,69 @@ private static Result exec(String... args) throws IOException {
...
+    */
+    private static String getKeystore(Project project) throws IOException {
+        String keystore = project.option("keystore", "");
+        if (keystore.length() == 0) {
+            keystore = "debug.keystore";
+            String keystoreAlias = "androiddebugkey";
+            String keystorePassword = "android";
+            String keystorePasswordFile = "debug.keystore.pass.txt";
+            if (!new File(keystore).exists()) {
+                Result r = exec(getJavaBinFile("keytool"),
```
Comment (JCash) [inline com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidBundler.java:141]: 👍

**Review by JCash — APPROVED** [review]

**Pull request merged**